ocs.tex

```diff
@@ -0,0 +1,80 @@
+\section{Observatory Control System}\label{sect:ocs}
+The LSST Observatory Control System consist of a collection of specialized components, namely; LOVE, the ScriptQueue, the Watcher and Control Systems. This distributed control system is designed to efficiently and safely perform astronomical observations individually or through automated scheduling. In this section we describe the role each of those components play in enabling the LSST observatory operations. 
+
+\subsection{The ScriptQueue component} \label{sect:scriptq}
+There are a number of different ways users can interact with components in the LSST system. For instance, one could easily use the SAL generated API in any of the supported languages to send commands directly to a single or multiple components. It is also possible to use SalObj Remotes to write Python scripts (e.g. SAL Scripts) that would command different components to accomplish a specified task. Not to mention that LOVE itself provides a customizable interface for users to interact with components.
+
+During commissioning and operations the LSST system will require a high degree of coordination between different crews (different daytime and nighttime shifts, for instance), not to mention the increasing number of available components and level of complexity as the system ramps up. In order to manager those issues, the LSST control system contains a specialized script queuing component, a.k.a. the ScriptQueue\footnote{\url{https://github.com/lsst-ts/ts_scriptqueue}}.
+
+The ScriptQueue defines an interface for developing scripts in general, and provides BaseScript, a base class for Python SAL scripts. As Python programs, these scripts have access to all Python functionality, both from the native Python 3 language and through imported modules (including \asyncio~to manage concurrent activities or libraries from the DM stack). In particular, a SAL Script has access to all the system components using SalObj Remotes (\secref{sect:salobj}) . Although Python is the only language officially supported, scripts can be written in any SAL-supported language. As long as they follow the interface defined by the ScriptQueue component, it should be possible to execute them.
```

Comment:
Consider: "defines an interface for developing scripts in general" -> "defines and interface for SAL scripts" (at the minimum please add "SAL" before "scripts")

Space before period